lang/scala/scala.py

```diff
@@ -0,0 +1,269 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+user.tag: scala
```

Comment:
This should be `tag: user.scala`

Reply:
Arg! That'll teach me to smash it out on a break and not test it 🚯